scripts/qiita

```diff
@@ -30,6 +30,10 @@ from qiita_db.commands import (load_sample_template_from_cmd,
                                load_prep_template_from_cmd,
                                load_parameters_from_cmd, SUPPORTED_PARAMS,
                                update_preprocessed_data_from_cmd)
+from qiita_db.portal import (add_studies_to_portal,
```

Comment:
If the configuration doesn't specify the QIITA portal, this import will fail. Also, the commands shouldn't be defined if the portal is not QIITA.